Library/Homebrew/service.rb

```diff
@@ -162,8 +172,8 @@ def to_systemd_unit
       options << "StandardOutput=append:#{@log_path}" if @log_path.present?
       options << "StandardError=append:#{@error_log_path}" if @error_log_path.present?
       if @environment_variables.present?
-        list = @environment_variables.map { |k, v| "#{k}=#{v}" }.join("&")
-        options << "Environment=\"#{list}\""
+        list = @environment_variables.map { |k, v| "#{k}=#{v}" }
+                                     .each { |value| options << "Environment=\"#{value}\"" }
```

Comment:
This is a bit more concise, though feel free to pass if you think it's more confusing:

```suggestion
        options += @environment_variables.map { |k, v| "Environment=\"#{k}=#{v}\"" }
```